rpm/sailfish-setup.spec

```diff
@@ -47,6 +50,10 @@ groupadd -rf sailfish-datetime || :
 
 groupadd -rf timed || :
 
+groupadd -rf sailfish-calls || :
```

Comment:
Maybe this should be called "sailfish-phone" or "sailfish-voicecall"

Reply:
I chose the sailfish-phone, although I know it's not that great name to limit things on lower level but it mirrors sailfish-messages nicely. sailfish-voicecall would mean we'd need to call sailfish-messages to sailfish-sms or something but they wouldn't have connection to the applications. Hard to decide although it's such a small thing.